example_yamls/example_msdnet_customdil.yaml

```diff
@@ -8,15 +8,16 @@ io_parameters:
   mask_tiled_api_key: 
   seg_tiled_uri: http://0.0.0.0:8888/api/v1/metadata/mlex_store/rec20190524_085542_clay_testZMQ_8bit/results
   seg_tiled_api_key: 
-  uid: uid0022
+  uid_save: 
+  uid_retrieve: 
 
 model_parameters:
   network: "MSDNet"
   num_classes: 3
   num_epochs: 3
   optimizer: "Adam"
   criterion: "CrossEntropyLoss"
-  weights: [1.0, 2.0, 4.0]
+  weights: "[1.0, 2.0, 0.5]"
   learning_rate: 0.1
```

Comment:
Are these parameters the recommended default values within dlsia? I think the learning rate should be 0.01

Reply:
This applies to all the example yaml files

Reply:
I agree with 0.01, this is meant to quick testing on my cpu.

Going further on these example yaml files, I originally designed this to show users a template what to expect in different models cases, and a lot of parameters here are not meant for default recommended value but for quick self-testing purpose. I would suggest once I discussed with @Wiebke about default values in the UI and we wrap up the MVP, I will change everything here accordingly.